### ISSUE TITLE
Add support for contextual (open) entities for csv2json direction

### ISSUE THREAD

**Issue by balatjan** [IBM/watson-assistant-workbench#246]
Implement conversion from WAW inline csv intent notation
`I'd like <color>red</color> dress`
to WA json structure
```
        {
          "text": "I'd like red dress",
          "mentions": [
            {
              "entity": "color",
              "location": [
                9,
                12
              ]
            }
          ]
        }
```

**Comment by balatjan** [comment]
it would be also useful to check, if the entity for which you annotate exists. If it doesn't create a warning. Watson assistant will anyway refuse to upload such a workspace where annotation exists for non-existent entity.

**Comment by Tereza-Pytelova** [comment]
Csv2json conversion done and merged, rest will be done in #301
